gato/policy/gato_policy.py

```diff
@@ -117,12 +117,12 @@ def __init__(
         self.separator_token = nn.Parameter(torch.zeros(embed_dim))
 
         # Tokenizers
-        self.text_tokenizer = None # e.g. SentencePiece
+        self.text_tokenizer = TextTokenizerGPT2()
```

Comment:
can this be made configurable?

Reply:
Changing to AutoTokenizer since huggingface basically contains all major tokenizers neatly wrapped in an AutoTokenizer class. No point to duplicate.